pkg/upgrade/upgrade.go

```diff
@@ -328,7 +328,7 @@ func ensureAirgapArtifactsInCluster(ctx context.Context, cli client.Client, in *
 		return fmt.Errorf("ensure autopilot plan: %w", err)
 	}
 
-	nsn := types.NamespacedName{Name: "autopilot"}
+	nsn := types.NamespacedName{Name: "autopilot-artifacts"}
```

Comment:
this cannot be named the same as the upgrade plan as ec will never try to upgrade if it finds a plan with the same name https://github.com/replicatedhq/embedded-cluster-operator/pull/265/files#diff-73305974932187e897bdd2bbc1da8d15b4eba7313a3630833a75ac89a2851b1dR359